.travis.yml

```diff
@@ -28,13 +28,15 @@ branches:
 
 before_install:
   - gem install coveralls-lcov
-  - pip install coveralls
+  - pip install coveralls flake8 black
   - nvm install $TRAVIS_NODE_VERSION
   - npm install -g bower
   - npm install -g grunt-cli
   - npm install -g karma-cli
 
 install:
+  - flake8 --max-complexity=21  ./src ./test
```

Comment:
Same question as above, what is the reasoning behind the `--max-complexity` configuration value?

Reply:
@eamahanna because that was the highest complexity file I encountered. We could also change it to ignore complexity or set it lower, but if we set it lower we would have to start editing functions, which isn't in scope for this issue I believe

Reply:
Do you think it is important to configure the max-complexity value at this point? Has complexity within KBase been an issue?

This is just a nit pit. I generally prefer going with default configs on linters and adjusting them as needed. <- That's just my perspective though :)

Reply:
I don't have any strong feelings either way. I just wanted flake8 to pass for the current set of code. Codacy and SonarCloud will complain about max complexity for new PRs and it's an opportunity to look at your code and refactor it then.